main.go

```diff
@@ -78,11 +83,28 @@ func main() {
 		log.Fatalf("watch error: %v", err)
 	}
 
-	fmt.Println(snc.Hash())
+	var cmd *exec.Cmd
```

Comment:
what is the reason to re-use this pointer?

Reply:
The same logic as explained in above comment.
"I need *exec.Cmd object to save in the main function. In order to track that if the old process is already running or not. Exec doesn't return *exec.Cmd object."

Reply:
in that case please start a new goroutine and use a channel to send updates to this goroutine which publishes

Reply:
Okay